lib/src/lib/components/form-container/form-container.component.ts

```diff
@@ -71,24 +71,48 @@ export class Lab900Form<T> implements OnChanges {
       );
     }
     if (!changes?.data?.isFirstChange() && this.data) {
-      setTimeout(() =>
-        this.patchValues(this.data, changes?.data?.previousValue)
-      );
+      setTimeout(() => this.patchValues(this.data));
     }
   }
 
-  public patchValues(data: T, prevData?: T): void {
+  public patchValues(data: T): void {
     Object.keys(data).forEach((key: string) => {
       const control = this.form.controls[key];
-
-      if (control && isDifferent(data[key], prevData?.[key])) {
```

Comment:
Why do you no longer check if there's any difference in the data?

Reply:
So what is actually going on is this:
data = initial form data, it does not update when editing a form field, or when adding or deleting a row.
This means you can make changes, and want to reset the form, but it wont happen, because the reset data is still the same as the initial data.

because of my changes this is also ok, because we no longer re-initialize the whole form